docs/workflows/steps/params/README copy.md

```diff
@@ -0,0 +1,103 @@
+# Params
```

Comment:
I realized this copy and the other README are both included - did you mean to include this file in the commit?